python-main.js

```diff
@@ -463,7 +502,7 @@ function web_editor(config) {
         // Focus on the element with TAB-STATE=1
         $("#command-download").focus();
     }
-
+    
```

Comment:
I might have left this behind, could you remove this one as well?

Reply:
Done.